Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -19,6 +19,17 @@ class PostRideSummaryViewController: UIViewController {
     private let locationManager = CLLocationManager()
     private let mapView = MKMapView()
     
+    private var ride: Ride!
```

Comment:
make this optional instead?

Reply:
We are guranteed to always pass in a ride when we initalize the view controller so this cleans things up.